airflow/providers/google/cloud/hooks/vertex_ai/generative_model.py

```diff
@@ -0,0 +1,133 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module contains a Google Cloud Vertex AI Generative Model hook."""
+
+from __future__ import annotations
+
+from typing import Sequence
+
+import vertexai
```

Comment:
Can you please tell what is this package? I can't find it in providers.yaml file and i think for vertexAI we are using google-cloud-aiplatform, no?

Reply:
Got it!
Thank you for explanation :)
So, this `vertexai` is available only for google-cloud-aiplatform>=1.41.0, right?

Reply:
Correct. somewhat confusing, there is an outdated (May 2023) pypi package [vertexai](https://pypi.org/project/vertexai/), but the project description indicates: To use the Vertex AI SDK, please install the google-cloud-aiplatform PyPi package by running pip install google-cloud-aiplatform.